pkg/routeagent-driver/handlers/kubeproxy-iptables/kp-iptables.go

```diff
@@ -0,0 +1,107 @@
+package kp_iptables
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"sync"
+
+	"k8s.io/klog"
+
+	cableCleanup "github.com/submariner-io/submariner/pkg/cable/cleanup"
+	clientset "github.com/submariner-io/submariner/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner/pkg/event"
+	"github.com/submariner-io/submariner/pkg/routeagent-driver/cni_interface"
+	"github.com/submariner-io/submariner/pkg/routeagent-driver/constants"
+	"github.com/submariner-io/submariner/pkg/routeagent/cleanup"
+	"github.com/submariner-io/submariner/pkg/util"
+)
+
+type SyncHandler struct {
+	event.HandlerBase
+	clusterID        string
+	namespace        string
+	localCableDriver string
+	localClusterCidr []string
+	localServiceCidr []string
+
+	remoteSubnets    *util.StringSet
+	remoteVTEPs      *util.StringSet
+	routeCacheGWNode *util.StringSet
+
+	syncHandlerMutex     *sync.Mutex
```

Comment:
This does not need to be a pointer - avoids having to initialize.

```suggestion
	syncHandlerMutex     sync.Mutex
```